djangocms_blog/models.py

```diff
@@ -171,7 +172,8 @@ class Post(KnockerModel, ModelMeta, TranslatableModel):
 
     translations = TranslatedFields(
         title=models.CharField(_('title'), max_length=255),
-        slug=models.SlugField(_('slug'), max_length=255, blank=True, db_index=True),
+        slug=models.SlugField(_('slug'), max_length=255, blank=True,
```

Comment:
To make it compatible for 1.8+ we can use following library 
https://github.com/neithere/django-autoslug

What do you think ?